src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -105,6 +105,21 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
   const projectOptions: { label: string; id: string }[] = [];
   const wbsDropdownOptions: { label: string; id: string }[] = [];
 
+  const customComparator = (option1: { id: string }, option2: { id: string }) => {
+    // Extract the WBS numbers from the options and convert them to a sortable format
+    const wbsNumber1 = parseFloat(option1.id.replace(/-/g, '.'));
+    const wbsNumber2 = parseFloat(option2.id.replace(/-/g, '.'));
+
+    // Compare the WBS numbers and return -1, 0, or 1 based on the comparison
+    if (wbsNumber1 < wbsNumber2) {
+      return -1;
+    } else if (wbsNumber1 > wbsNumber2) {
+      return 1;
+    } else {
+      return 0;
+    }
```

Comment:
this logic may technically work, but we should compare the elements of a wbsNum (car number, project number, and work package number) individually, to make sure we're being more precise. You can do this by calling .split('.') on each wbsNum string to get an array of strings [carNumber, projectNumber, wpNumber]. You can then compare the carNumber1 to carNumber2 and return -1 or 1 if one is greater/less or go on to check the projectNumber if they're equal (and so on)

Reply:
lmk if that's confusing

Reply:
You could also just use validateWbs function to get an object in that format and compare like reid said so u dont have to mess with split strings